.hooks/pre-commit.py

```diff
@@ -0,0 +1,22 @@
+import re
+import sys
+
+def main():
+    stage_files = sys.argv[1:]
+    for stage_file in stage_files:
+        conditions = []
+        with open(stage_file, 'r') as f:
+            file_contents = f.readlines()
+            if not file_contents:
+                continue
+            for line in file_contents:
+                if re.match('^[a-zA-Z]', line):
+                    conditions.append(line.split()[0].split(":")[0])
+            sorted_conditions = conditions
```

Comment:
This is not to copy `conditions` to `sorted_conditions`. This is just to assign the same reference to the same list to variable `sorted_conditions`. After `sorted_conditions` is sorted, the list referenced by `conditions` is changed too. Because these two variables are pointing to the same list.
You probably need to import the `copy` module to create a copy of the list.